src/main/java/org/eolang/dedup/DeDuplicate.java

```diff
@@ -0,0 +1,26 @@
+package org.eolang.dedup;
+
+import org.eolang.dedup.Main;
+import java.io.IOException;
+
+public class DeDuplicate {
+    /**
+     * Path to file to be transformed.
+     */
+    private final String path_input;
+
+    /**
+     * Path to output file.
+     */
+    private final String path_output;
+
+    public DeDuplicate(final String i_pth, final String o_pth) {

```

Comment:
@cahekska It's better to use `cammelCaseNames` instead of underscores. Please read that document about java code conventions: https://www.oracle.com/technetwork/java/codeconventions-150003.pdf